app/templates/search/filter_list.html

```diff
@@ -1,47 +1,122 @@
 
+{% from 'search/macros/filters/collections.html' import collections %}
 {%- set pageTitle = 'More filter options' -%}
-
 {% block stylesheets %}
-  {{ super() }}
-  <link rel="stylesheet" href="{{ static('catalogue-results.css') }}?v={{ app_config.BUILD_VERSION }}" media="screen,print">
+{{ super() }}
+<link rel="stylesheet" href="{{ static('catalogue-results.css') }}?v={{ app_config.BUILD_VERSION }}" media="screen,print">
 {% endblock %}
 
 {% block beforeContent %}
-  <div class="tna-background-accent">
-    <div class="tna-container">
+<div class="tna-background-accent">
+   <div class="tna-container">
       <div class="tna-column tna-column--full">
-        {{ tnaBreadcrumbs({
-          'items': [
-            {
-              'text': 'Home',
-              'href': '/'
-            },
-            {
-              'text': 'Catalogue',
-              'href': url('main:catalogue')
-            }
-          ],
-          'structuredData': True,
-          'classes': 'tna-!--padding-vertical-s'
-        }) }}
+         {{ tnaBreadcrumbs({
+         'items': [
+         {
+         'text': 'Home',
+         'href': '/'
+         },
+         {
+         'text': 'Catalogue',
+         'href': url('main:catalogue')
+         }
+         ],
+         'structuredData': True,
+         'classes': 'tna-!--padding-vertical-s'
+         }) }}
       </div>
-    </div>
-  </div>
-{% endblock %}
+   </div>
+   <div class="tna-container">
+      <h2 class="tna-heading-l tna-heading--long">Select filters to apply to your results</h2>
+   </div>
+</div>
+<div class="tna-container tna-section tna-section--long">
+   <a href="#" class="tna-button">
+   Apply Filters
+   </a>
+   <a href="#" class="tna-column tna-column--width-1-4 tna-column--width-1-3-medium tna-column--width-1-2-small tna-column--full-tiny">
+   Cancel and return to results
+   </a>
+</div>
+
+<div class="tna-container">
+   <div id="filter-panel" class="tna-column tna-column--width-1-3 tna-column--width-1-3-medium tna-column--full-small tna-column--full-tiny tna-!--margin-top-m ">
+    <hr>
+      <div class="tna-aside tna-aside--tight tna-background-tint tna-!--margin-top-s tna-background-accent-light" style="display: block;">
+         <div class="tna-form__group " data-module="tna-text-input">
+            <div class="tna-form__group-contents">
+               <h3 class="tna-form__heading tna-form__heading--s">
+                  <label class="tna-form__label" for="search-within">
+                  Search within filters
+                  </label>
+               </h3>
+            </div>
+            <input id="search-within" class="tna-text-input " name="search_within" value="" type="text" spellcheck="false" autocapitalize="off" autocorrect="off" form="search-form">
+            <p>Already selected filters will remain active.</p>
+         </div>
+      </div>
+   </div>
+
 
+
+<div id="results" class="tna-column tna-column--width-2-3 tna-!--margin-top-l">
```

Comment:
```suggestion
   </div>

   <div id="results" class="tna-column tna-column--width-2-3 tna-!--margin-top-l">
```
1. There are multiple blank lines here; let’s reduce it to one between div's for consistency.
2. There seems to be a missing closing div here. I counted 13 openings and only 12 closings.